Tetris/src/tetris/GameDisplay.java

```diff
@@ -0,0 +1,49 @@
+package tetris;
+
+/**
+ * Class for displaying game elements to the screen.  
+ * 
+ * This will be driven by the GameRunner with data retrieved from GameSystem
+ * 
+ * This is going to have some fields but we haven't worked that out yet
+ */
+public class GameDisplay {
+	//TODO: add fields, such as String flavorText, actual display, etc.
+	
+	/**
+	 * Default constructor
+	 * initialize default display
+	 */
+	public GameDisplay(){
+		
+		// Game board is 22 blocks high by 10 blocks wide
+		
+		// Game board has a border
+		
+	}
+	
+	/**
+	 * Update display
+	 * @param gridData
+	 */
+	public static void updateGridDisplay(boolean[][] gridData){
+		
+	}
+	
+	/**
+	 * Update flavor text (pause, quit, etc)
+	 * @param text
+	 */
+	public static void updateFlavorText(String text){
```

Comment:
@tatiana5kova 
I imagined flavor text could be "pause", "quit", etc. but we could have another method for updating text from darth tetro